src/utils/countly.js

```diff
@@ -2,6 +2,8 @@ import settings from './settings'
 
 export const events = {
   CODE_RESET: 'resetCode',
+  CODE_VIEW_SOLUTION: 'viewSolutionCode',
+  SUCCESS_LINK_CLICK: 'successLinkClick',
```

Comment:
@terichadbourne would this work?

```suggestion
  LINK_CLICK_IPLD_EXPLORER: 'linkClickIpldExplorer',
  LINK_CLICK_CID_INSPECTOR: 'linkClickCidInspector',
```